package.json

```diff
@@ -101,11 +101,5 @@
     "webpack-node-externals": "1.5.4",
     "wholesale-styleguide-scss": "git+https://github.com/wholesale-design-system/scss.git#master",
     "yargs": "7.0.2"
-  },
-  "peerDependencies": {
```

Comment:
Pls install all dependencies such React and Preact only as peer. Otherwise NPM will install this as production dependencies even If I don't use it. If you use `npm i -D preact` it will install them as dev deps and you have to manually put them to peer.

Reply:
Good point. I'll look into it on Monday (I'm on vacation now).

BTW, what do you think about moving some other deps, such as sinon, eslint into `devDependencies`? They shouldn't be production dependencies either. (I would do such cleanup in a separate PR, since that's out of scope for this one.)

Reply:
So have a nice time :)
Actually all dependencies should be installed with -D flag for devDeps. Only react, preact and fabric-scss should be installed as peerDependencies since this packages are optionally needed in production.